ambry-api/src/main/java/com/github/ambry/config/ReplicationConfig.java

```diff
@@ -63,6 +63,14 @@ public class ReplicationConfig {
   @Default("1")
   public final int replicationNumOfInterDCReplicaThreads;
 
+  /**
+   * If true, replication Get requests asks for deleted and expired blobs as well to succeed in scenarios where blobs
+   * get deleted or expired after replication metadata exchange.
+   */
+  @Config("replication.use.separate.thread.for.each.host")
```

Comment:
use public static final in CAPS for this string replication.use.separate.thread.for.each.host